### PR TITLE
Geolocation implementations #5

### PR DESCRIPTION
**Pull request by trucidare**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
```
Comment (PPetau) [inline .vscode/settings.json:1]: Can we ignore the configuration from the dotnet extension?

File: MauiBundler/Plugins/Plugin.Geolocation/wwwroot/Geolocation.cs.js [hunk 2]
```diff
@@ -1,3 +1,31 @@
+var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (receiver, state, kind, f) {
```
Comment (PPetau) [inline MauiBundler/Plugins/Plugin.Geolocation/wwwroot/Geolocation.cs.js]: This should not be included

**Review by PPetau — CHANGES_REQUESTED** [review]
Looks fine. Just some files that should not end up in the repository

File: .vscode/settings.json (hunk 1 above)
Comment (trucidare) [reply .vscode/settings.json:1]: Yes - removed now

File: MauiBundler/Plugins/Plugin.Geolocation/Geolocation.cs.ts [hunk 3]
```diff
@@ -1,6 +1,35 @@
+export class GeoPosition {
```
Comment (PPetau) [inline MauiBundler/Plugins/Plugin.Geolocation/Geolocation.cs.ts]: This should be an Interface only

Comment (trucidare) [reply MauiBundler/Plugins/Plugin.Geolocation/Geolocation.cs.ts]: Done

File: MauiBundler/Plugins/Plugin.Geolocation/Services/GeolocationService.android.cs [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    public void OnStatusChanged(string? provider, [GeneratedEnum] Availability status, Bundle? extras)
+    {
+        Task.Run(async () => await PublishStatusChangedEvent($"{provider} change his status and now it's {status}"));
+    }
+
+    private async Task PublishStatusChangedEvent(string message)
+    {
+        await jsRuntime.InprocessJSRuntime!.InvokeVoidAsync($"MauiBundler.Plugins.{nameof(GeoLocation)}.statusChanged", message);
```
Comment (PPetau) [inline MauiBundler/Plugins/Plugin.Geolocation/Services/GeolocationService.android.cs:83]: This can be something we should handle internally by the Bundler

**Review by PPetau — APPROVED** [review]
LGTM

**Pull request merged**